.nais/application/application-config-prod-gcp.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: "nais.io/v1alpha1"
+kind: "Application"
+metadata:
+  name: veilarbveileder
+  namespace: obo
+  labels:
+    team: obo
+spec:
+  image: {{image}}
+  port: 8080
+  webproxy: true
+  secureLogs:
+    enabled: true
+  liveness:
+    path: veilarbveileder/internal/isAlive
+    initialDelay: 30
+  readiness:
+    path: veilarbveileder/internal/isReady
+    initialDelay: 30
+  prometheus:
+    enabled: true
+    path: veilarbveileder/internal/prometheus
+  replicas:
+    min: 2
+    max: 4
+    cpuThresholdPercentage: 75
+  resources:
+    limits:
+      cpu: "2"
+      memory: 2048Mi
+    requests:
+      cpu: 500m
+      memory: 1024Mi
+  ingresses:
+    - https://veilarbveileder.intern.nav.no
```

Comment:
ingress should not be the same in fss and gcp (I think)

Reply:
done :D